### ISSUE TITLE
Ability to add full address to Noreply-Extension

### ISSUE THREAD

**Issue by woolmilkpigs** [freescout-helpdesk/freescout#1794]
Hello!

As some incoming mails shouldn't get an auto reply I use the extension, to not send any auto replies to incoming mails with a certain prefix before the @ sign. Is there an ability - or plans to be able - to add full address including domain to that no-auto-reply-list? As certain companies use prefixes which wont handle auto replies, but others do (fex. service@paypal.com).

Thanks!

**Comment by freescout-helpdesk** [comment]
Feel free to add a feature request: https://freescout.net/request-feature/

**Comment by woolmilkpigs** [comment]
Meanwhile I looked up the code of the plugin and noticed, that the script checking for a match is a simple regex which is looking for any match of the given string in the mail address of potentially autoreplied mails. As the settings page doesn't hinder me from typing an whole address, the system works already as I suggested :-)

So maybe updating the doc on the setting page?

Instead of 

"Enter email prefixes without @ sign one per line. Use a dash where you want to allow a dash or no symbol."

something like

"Enter any mail address (prefix, domain or wohle address) per per line ...."
